base_classes/NXinstrument.nxdl.xml

```diff
@@ -42,6 +42,31 @@
 	        <doc>short name for instrument, perhaps the acronym</doc>
 	    </attribute>
 	</field>
+    <field name="energy_resolution" type="NX_FLOAT" units="NX_ENERGY">
```

Comment:
To us, these resolution entries also felt a bit out of place here, because NXinstrument just contained base class links before, and we cannot be sure to cover all possible cases. Maybe a generic resolution base class, that you can include as often as necessary is the better way to go.